core/zoe/zoe.chainmail

```diff
@@ -0,0 +1,163 @@
+/**
+ * This is returned by a call to `makeInstance` on Zoe.
+ */
+struct UserFacetAndGoverningContract ( ) {
+  zoeInstance :ZoeUserFacet;
+  governingContract :CustomGoverningContract;
+}
+
+/**
+ * This is returned by a call to `escrow` on the zoeInstance userFacet.
+ * An escrowReceipt is an ERTP payment which acts as proof of escrow
+ * and is normally passed in to a governingContract method. A
+ * claimPayoff ERTP payment is what can be unwrapped to claim the
+ * payoff of involvement with a contract. 
+ */
+struct UserFacetAndGoverningContract ( ) {
+  escrowReceipt :Payment;
+  claimPayoff :Payment;
+}
+
+/**
+ * Zoe is a long-lived and well-trusted contract that enforces
+ * offer-safety for the contracts that use it. Zoe has a single
+ * `seatIssuer` and a single `escrowReceiptIssuer` for the entirety of its
+ * lifetime. By having a reference to Zoe, a user can get the `seatIssuer`
+ * or `escrowReceiptIssuer` and thus validate any `seat` or `escrowReceipt`
+ * they receive from someone else. 
+ */
+
+
+interface Zoe ( ) {
+  /** 
+   * Zoe is long-lived, but we want to create individual zoeInstances
+   * that are instantiated with a particular governingContract and
+   * particular issuers. For example, a zoeInstance may be instantiated
+   * with a swap contract and an array of the issuers for moola and
+   * simoleans. We get back an object that has the user facet for the
+   * zoeInstance and the governing contract facet. `makeContract` is a
+   * function that takes a zoeInstance
```

Comment:
Add a period at the end so it doesn't look like we were thinking about saying more.